administration/src/project-configs/koblenz/dataPrivacyBase.tsx

```diff
@@ -1,6 +1,6 @@
 import { ReactElement } from 'react'
 
-export const dataPrivacyBaseHeadline = 'Datenschutzerklärung für die Nutzung und Beantragung des digitalen Koblenz-Pass'
+export const dataPrivacyBaseHeadline = 'Datenschutzerklärung für die Nutzung und Beantragung des digitalen KoblenzPass'
```

Comment:
we also have 'Koblenz-Pass' in PDF, should it be fixed there as well?

Reply:
hm strange for me there is "KoblenzPass" i already adjusted this

Reply:
where? this is the pdf from your branch:
https://github.com/digitalfabrik/entitlementcard/blob/28b349a5afdd59fcb34113a0f6b405a56ada535b/administration/src/project-configs/koblenz/pdf-template.pdf
I mean the first line with "Guten Tag..." and with "Laden Sie sich die App ..."